openshift-ci/build-root/OWNERS

```diff
@@ -0,0 +1,20 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- sbose78
+- wtam2018
+- bigkevmcd
+- chetan-rns
+
+reviewers:
+- sbose78
+- wtam2018
+- bigkevmcd
+- chetan-rns
+- mitchharpur
+- keithchong
+- tetchel
+- amitkrout
+- jaideepr97
+- gaganhegde
```

Comment:
pls remove folks who have left. mitchharpur, jaideepr97, gaganhegde and add new folks

Reply:
Done. Thanks for the note.